packages/suite/src/components/wallet/SearchAction.tsx

```diff
@@ -102,13 +109,9 @@ export const SearchAction = ({
                 size="small"
                 innerRef={inputRef}
                 innerAddon={
-                    <Icon
-                        name="search"
-                        cursorPointer
-                        size={16}
-                        color={theme.iconDefault}
-                        onClick={!isExpanded ? openAndSelect : undefined}
-                    />
+                    <SearchIconContainer onClick={!isExpanded ? openAndSelect : undefined}>
+                        <Icon name="search" cursorPointer size={16} color={theme.iconDefault} />
```

Comment:
Could you please add variant="default" to the Icon component with this color so you can use variant instead? See the Text component, it works the same.